Autopilot/DataStructures/Src/vectorClass.cpp

```diff
@@ -0,0 +1,108 @@
+/* 
+    Author: Nixon Chan
+*/
+
+#include <math.h>
```

Comment:
math.h is deprecated in C++, should use cmath. You'll have to call std::sqrt() once you do.